en/editor/inspector/index.md

```diff
@@ -1,103 +1,112 @@
+Developer scripts in the **Assets** panel can be dragged and dropped directly into the **Inspector** panel to generate a script component, or added via **Add Component -> Custom Scripts**. The properties of a script component are declared by the script. Different types of properties have different control appearance and editing in the **Inspector**. We will add the properties in [Declare Properties](... /... /scripting/ccclass.md) section for details on how properties are defined.
 
-  ![ui-color](index/ui-color.png)
+## Attribute Types
 
-### Reference type properties
+**Properties** are variables declared in component scripts that are public and can be serialized and stored in scene and animation data. The **property inspector** allows you to quickly modify property settings for the purpose of adjusting game data and gameplay without modifying the script.
+
+Properties can usually be divided into two main categories, **value types** and **reference types**, depending on where the variables use memory.
 
-**Reference types** include objects, such as **nodes**, **components**, or **assets**. You can select and assign values ​​by **dragging** or **popping up the Assets panel**.
+### Value type attributes
 
-![assets-panel](index/assets-panel.png)
+**Value types** include simple variable types that take up very little memory, such as numbers, strings, booleans, enumerations, etc.
 
-## Batch operationas
+- `Numeric (Number)`: can be entered directly using the keyboard, or the property value can be incremented or decremented by pressing the up and down arrows next to the input box.
+- `Vector (Vec2)`: The control of a vector is a combination of two numeric inputs, and the input box is marked with x and y to identify the sub-property name corresponding to each value.
+- `String`: Enter a string directly into the text box using the keyboard.
+- `Boolean`: Edit in the form of a checkbox, the selected state means the property value is true, the non-selected state means false.
+- `Enum`: edit in the form of a drop-down menu, click the enumeration menu, and then select an item from the pop-up menu list to finish modifying the enumeration value.
+- Color: Click on the color property preview box, the **Color Picker** window will pop up, in this window you can use the mouse to directly click on the desired color, or directly enter the specified color in the RGBA color input box below. Clicking anywhere outside the **Color Picker** window will close the window and use the last selected color as the property value. For example, the color picker component.
```

Comment:
- `Color`: Click on the color property preview box, the **Color Picker** window will pop up, in this window use the mouse to directly click on the desired color, or directly enter the specified color in the RGBA color input box below. Clicking anywhere outside the **Color Picker** window will close the window and use the last selected color as the property value. For example, the color picker component.